src/lib.rs

```diff
@@ -62,8 +65,15 @@ where
 
     /// The root account represents the public/private key as it's returned by the vault.
     /// It's recommended to create sub-accoutns and used those instead.
-    pub fn root_account(&self) -> Result<&Account<V::Pair>> {
-        self.root.as_ref().ok_or(Error::Locked)
+    pub fn root_account(&self) -> Result<Rc<Account<V::Pair>>> {
+        self.root.clone().ok_or(Error::Locked)
+    }
+
+    pub fn create_sub_account<'a>(&mut self, name: &str, derivation_path: &str) -> Result<&Account<V::Pair>> {
```

Comment:
Not sure the lifetime declaration si needed

Reply:
Changed in commit https://github.com/virto-network/libwallet/pull/15/commits/9f1433b9d225ef528d72fce3c76f4eb1b17293d8